base:
  base_manager:
    ros__parameters:
      status_topic: 'status'
      bay_id: [1,2,3,4,5,6,7,8]
      host: 'localhost'
      port: 27017
      database: 'passform'
      bay_db: 'bays'
      module_db: 'modules'
      basyx:
        registry_port: 8082
        registry_type: REST

datastore:
  ros__parameters:
    host: 'localhost'
    port: 27017
    database: 'passform'
